spec/manual/sni_required.cr

```diff
@@ -0,0 +1,10 @@
+require "spec"
+require "http"
+
+describe "HTTP::WebSocket uses SNI" do
+  host = "certstream.calidog.io"
+  it "should connect to #{host} with the websocket protocol" do
+    ws = HTTP::WebSocket.new "wss://certstream.calidog.io/"
```

Comment:
You can do `"wss://" + host + '/'`

Reply:
`"wss://#{host}/"`,  please

Reply:
We can get rid of the slash `/` - I don't know if `"wss://" + host` is better/worse than `"wss://#{host}"`

Reply:
The slash should stay, that's the normalized form. Interpolation should be preferred over concatenation, although the latter is a little bit faster for simple strings.